scripts/postbuild.js

```diff
@@ -122,10 +122,36 @@ function clearTempDir () {
   removeSync(tempFolder)
 }
 
+function copyFontsAssets () {
+  try {
+    copySync(
+      resolve(__dirname, '../scripts/assets'),
+      resolve(__dirname, '../dist/assets/')
+    )
+  } catch (err) {
+    console.log('Error copying fonts to dist')
+    console.log(err)
+  }
+}
+
+function copyFontsCss () {
```

Comment:
I like idea of keeping all separate, but I think that we can create one single general function 'copyAssets', that will copy everything from `./assets/` to `./dist/assets/`.